client-implementation.md

```diff
@@ -0,0 +1,67 @@
+# Client Implementations
+
+This document describes client implementation recommendations for interoperability with registry servers and other clients using the OCI Distribution Spec.
```

Comment:
The "other clients" comment comes from referrers fallback to a tag, since it involves multiple client tools modifying the same manifest that is referenced by a single tag. E.g. if the image signing tool assumes it's the only tool pushing artifacts and it overwrites the current tag, that would effectively delete the SBOM and other artifacts associated with the image.